spec/factories/addresses.rb

```diff
@@ -10,5 +10,9 @@
     trait :is_lookup_used do
       lookup_used { true }
     end
+
+    trait :with_address_for_xml_fixture do
+      postcode { 'GH08NY' }
+    end
```

Comment:
This doesn't seem related to the rest of the PR?

Reply:
it is actually - it resolves a flickering test which other changes here made worse - I couldn't work out why. But this ensures that the applicant is created with the correct postcode for the xml fixture file.